UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment/Services/ExchangeSetApiService.cs

```diff
@@ -15,9 +15,9 @@ public class ExchangeSetApiService : IExchangeSetApiService
         private readonly IAuthTokenProvider _authTokenProvider;
         private readonly ILogger<ExchangeSetApiService> _logger;
 
-        public ExchangeSetApiService(ILogger<ExchangeSetApiService> logger,
-                                     IOptions<ExchangeSetApiConfiguration> exchangeSetApiConfiguration,
-                                     IExchangeSetApiClient exchangeSetApiClient, IAuthTokenProvider authTokenProvider)
+        public ExchangeSetApiService(ILogger<ExchangeSetApiService>? logger,
+                                     IOptions<ExchangeSetApiConfiguration>? exchangeSetApiConfiguration,
+                                     IExchangeSetApiClient? exchangeSetApiClient, IAuthTokenProvider? authTokenProvider)
         {
```

Comment:
What's the point of making these parameters nullable if the constructor then uses null coalescing assignments to throw an argument null exception? Why not just leave these parameters as not nullable?

Reply:
Resolved. Removed the nullable parameters.